### DISCUSSION TITLE
Not receiving verification email for a new account

### DISCUSSION THREAD

**Discussion by sanchivarma** [community/community#45135] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Bug

### Body

Hi Team, 

I am trying to make an account from a different email for someone. It has been an hour and no verification email was received. 
Is this a known issue ? Is there an approximate duration for wait ? 

Thankyou

**Comment by jge162** [comment]
It’s fast. You should have got it. Check spam and spelling of email.

**Reply by sanchivarma** [reply]
Hi, Yes the email is correct. No there is no spam or any other email.
I am speculating there's a captcha or some other sort of confirmation after Verify Your Account that is not loading on github.
Sharing the screenshot 
(No additional settings on browser)

![Screenshot (13)](https://user-images.githubusercontent.com/31968471/214116137-1b294f5b-f7af-47b4-8729-3dc494adfc85.png)

**Reply by sanchivarma** [reply]
Also tried in Incognito and Other browsers (Firefox). Same behaviour

**Comment by sanchivarma** [comment, marked as answer]
i was able to sign up from my mobile. It was the puzzle in the empty box below that is (still) not loading on my browser.

**Reply by jge162** [reply]
Glad u figured it out

**Comment by ypytomets** [comment]
Have a similar issue: do not receive verifying confirmation mail on my proton mailbox.
How that can be solved?

**Comment by nguy1544** [comment]
I am having a similar issue. I am trying to associate my personal github with org email but I am not receiving the verification email. I have tried to remove it and re-add the email. Still nothing in inbox and spam.

**Comment by vanschroeder** [comment]
I encountered this issue -- seems Zoho was putting this in some bucket called "notifications" and giving no notifications about anything.
So if you're a Zoho user, look in "notifications" not "spam" or "inbox". Something that GitHub is doing is causing all their messages to be sent to "notifications".
